<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="mplwindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplvl"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="mplfigs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionAnalyze"/>
    <addaction name="actionCon"/>
   </widget>
   <widget class="QMenu" name="menuOverlap_Peaks">
    <property name="title">
     <string>Overlap Peaks</string>
    </property>
    <addaction name="actionAsBr"/>
    <addaction name="actionAsPb"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuOverlap_Peaks"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="text">
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionNPA">
   <property name="text">
    <string>Net Peak Area</string>
   </property>
  </action>
  <action name="actionAR">
   <property name="text">
    <string>Auto Roi</string>
   </property>
  </action>
  <action name="actionCon">
   <property name="text">
    <string>Standard Concentration</string>
   </property>
  </action>
  <action name="actionAs_Pb">
   <property name="text">
    <string>As (Br)-Pb</string>
   </property>
  </action>
  <action name="actionAs_Br">
   <property name="text">
    <string>As-Br</string>
   </property>
  </action>
  <action name="actionAs_Br_2">
   <property name="text">
    <string>As-Br</string>
   </property>
  </action>
  <action name="actionAsBr">
   <property name="text">
    <string>As-Br</string>
   </property>
  </action>
  <action name="actionAsPb">
   <property name="text">
    <string>As(Br)-Pb</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
